src/SAWScript/Interpreter.hs

```diff
@@ -471,7 +471,9 @@ buildTopLevelEnv proxy opts =
                    , rwProfilingFile = Nothing
                    , rwLaxArith = False
                    , rwWhat4HashConsing = False
+                   , rwWhat4HashConsingX86 = False
```

Comment:
I think that enabling hash consing during simulation breaks an AES-GCM rewrite rule, but hash consing is needed for the `w4_unint` tactic in that proof. It might be better to redivide these into options for enabling hash consing during simulation and during tactic execution separately; I've added a `FIXME` mentioning this.